docroot/themes/custom/twentyeighteen/sass/components/footer/footer.twig

```diff
@@ -0,0 +1,21 @@
+<div class="footer">
+  <div class="footer-text-block">
+      <p class="footer-text">Drupal GovCon, a production of Drupal4Gov, is made possible by our wonderful sponsors and volunteers. Graciously hosted by the NIH Library.</p>
+  </div>
+  <div class="footer-links-block">
+    <div class="footer-social_links">
+      <a href="https://www.facebook.com/DrupalGovCon">
+        <img src="../images/facebook.svg" alt="F" width="32px" height="32px" />
+      </a>
+      <a href="https://www.twitter.com/DrupalGovCon">
+        <img src="../images/twitter.svg" alt="T" />
+      </a>
+      <a href="mailto:drupal4gov@gmail.com?subject=Contact from Website">
+        <img src="../images/email.svg" alt="E" />
+      </a>
+    </div>
+  </div>
+  <div class="footer-image-block">
```

Comment:
If we are using BEM structure here, it would probably be better to have the footer wrapper be the block level (already done) 
The next child I believe should be at the "Element" level. 

Take a look at this and try modifying it :) http://getbem.com/naming/. We want to be as consistent as possible with naming so any children of any parent element should follow the same BEM pattern. Feel free to ping me if you are confused.